docs/config.md

```diff
@@ -131,6 +131,39 @@ php_server [<matcher>] {
 }
 ```
 
+### Watching for file changes
+
+Since workers won't restart automatically on file changes you can also
+define a number of directories that should be watched. This is useful for
+development environments.
+
+```caddyfile
+{
+    frankenphp {
+        worker /path/to/app/public/worker.php
+        watch /path/to/app/sourcefiles
+    }
+}
+```
+
+The configuration above will watch the `/path/to/app/sourcefiles` directory recursively.
+You can also add multiple `watch` directives
+
+```caddyfile
+{
+    frankenphp {
+        watch /path/to/app/folder1             # watches all subdirectories
+        watch /path/to/app/folder2/*.php       # watches only php files in the app directory
+        watch /path/to/app/folder3/**/*.php    # watches only php files in the app directory and subdirectories
+    }
+}
+```
+
+Be sure not to include files that are created at runtime (like logs) into you watcher, since they might cause unwanted
+worker restarts.
+The file watcher is based on [go-fswatch](https://github.com/dunglas/go-fswatch) and shares the limitations of the 
+underlying fswatch library.
```

Comment:
Is there any? And maybe should we link to the original "fswatch" project instead (the docs maybe more relevant than the go bindings and the tool is quite popular)?

Reply:
Yeah there are some small limitations on windows and linux, but maybe they are neglegible, I'll adjust id 👍